technology/delegated-proof-of-stake-consensus.php

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="Delegated Proof-of-Stake is a fast, efficient, and flexible consensus algorithm that enables stakeholder voting on the blockchain for democratic decisions.">
-    <meta name="keywords" content="dpos, network, blockchain, consensus, bitshares, witness">
+    <meta name="keywords" content="block, witness, network, consensus, stakeholder, bitshares">
```

Comment:
"consensus" is used in description and page title. Added

Reply:
ah yes, ok